clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/HubConnection.java

```diff
@@ -20,6 +20,7 @@
     private static final String RECORD_SEPARATOR = "\u001e";
     private HubConnectionState connectionState = HubConnectionState.DISCONNECTED;
     private Logger logger;
+    private Action1<Exception> onClosedCallback;
```

Comment:
I just reused one of the `Action` types I defined earlier. Should I make a specific `OnClosedCallback `interface though? Seems unnecessary imo

Reply:
I really don't know what the Java Way(TM) is here :). It looks like there is a type called [`Consumer<T>`](https://docs.oracle.com/javase/8/docs/api/java/util/function/Consumer.html) which is basically `Action<T>`. Since Java types can't differ by generic parameter count alone, we still need our custom handlers for `.on` but we could just use `Consumer<T>` here I guess.